sunkit_pyvista/mapsequence_animator.py

```diff
@@ -0,0 +1,27 @@
+import time
+
+
+class SequenceAnimator(object):
+    def __init__(self, time, map_meshes, color_maps):
+        self.angle = 0
+        self.animation_state = False
+        self.start_time = 0
+        self.time = time
```

Comment:
What is the time module used for?

Reply:
time interval for each map to be displayed